apps/site/assets/ts/schedule/components/schedule-finder/ScheduleModalContent.tsx

```diff
@@ -86,7 +101,7 @@ const ScheduleModalContent = ({
     selectedDirection,
     input.date
   ]);
-  useAwaitInterval(updateData, 1000);
+  useAwaitInterval(updateData, 10000);
```

Comment:
Thanks! Not sure how many folks come to Dotcom for down-to-the-second updates on the ground but I'm sure this'll suffice.
I hadn't had time to dig this up earlier, but now looking at `Site.TransitNearMe.format_prediction_time`, which formats the output times, we say "arriving" when a bus is < 60 seconds or a subway is < 30 seconds away. Which is to say we'd need an interval sufficiently small to be informative in those cases... which is to say 10 seconds will probably be great middle ground between polling too eagerly and not eagerly enough. Thanks!

Reply:
+1 to switching from every second to every 10 seconds seeming like a reasonable middle ground